### ISSUE TITLE
Issue with the process: EXTERNAL_TOOLS_IMPORT

### ISSUE THREAD

**Issue by marissaDubbelaar** [nf-core/epitopeprediction#183]
### Description of the bug

There seems to be an error when the netmhcpan version 4.0 is used.
I checked the ` ~/.nextflow/assets/nf-core/epitopeprediction/assets/external_tools_meta.json`  and this seems to be in there. Maybe the issue can be easily resolved?

```
"netmhcpan_darwin": {
        "4.0": {
            "version": 4.0,
            "software_md5": "042d16b89adcf5656ca4deae06b55cc6",
            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCpan-4.0/data.Darwin.tar.gz",
            "data_md5": "b4eef3c82852d677a98e0f7d753a36e2",
            "binary_name": "netMHCpan"
        },
        "4.1": {
            "version": 4.0,
            "software_md5": "5f6eab43feb80a24e32eb02b22e9db5f",
            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCpan-4.1/data.tar.gz",
            "data_md5": "4bdd3944cb4c5b8ba4d8900dae074c85",
            "binary_name": "netMHCpan"
        }
    },

```

### Command used and terminal output

```console
nextflow run nf-core/epitopeprediction --input input.csv -profile docker --max_memory 4.GB --max_cpus 4 --min_peptide_length 8 --max_peptide_length 12 --outdir ./ligandomat_results --netmhcpan_path /Users/dubbelaar/lib/netMHCpan-4.1b.Darwin.tar.gz
```

### Relevant files

[nextflow.log](https://github.com/nf-core/epitopeprediction/files/10139836/nextflow.log)

### System information

- Nextflow version 22.10.1
- Hardware: MacBook
- Executor: local
- Container engine: Docker
- OS: macOS
- Version of nf-core/epitopeprediction: 2.1.0

**Comment by christopher-mohr** [comment]
Hi Marissa, thanks for the report. I assume you also added `netmhcpan-4.1` to the `--tools` parameter value? Could you please try adding `--netmhc_system darwin`?

**Comment by marissaDubbelaar** [comment]
oops yes the `--tools` parameter included `syfpeithi,netmhcpan-4.1`. I just tested it out at it seems to work.
Thank you @christopher-mohr, close the issue if you like. But I was wondering maybe good to alter the version under `netmhcpan_darwin`, `4.0`?

**Comment by christopher-mohr** [comment]
> oops yes the `--tools` parameter included `syfpeithi,netmhcpan-4.1`. I just tested it out at it seems to work.
> Thank you @christopher-mohr, close the issue if you like. But I was wondering maybe good to alter the version under `netmhcpan_darwin`, `4.0`?

Yes, I noticed that as well, I will fix it in the next PR. Thanks!
